data/search/variables/language.variable

```diff
@@ -0,0 +1,11 @@
+en,en => in english
+en,en => english
+ru,ru => по русски
+ru,ru => на русском
+ru,ru => русский
+ru,ru => русски
+ru,ru => русском
+he,he => בעברית
+he,he => עברית
+es,es => en español
+es,es => español
```

Comment:
Please add for each language the term inside the brackets. E.g. (בעברית)

Reply:
done